kiliautoml/mixins/_hugging_face_mixin.py

```diff
@@ -0,0 +1,63 @@
+from abc import ABCMeta
+import os
+
+from transformers import (
+    AutoModelForSequenceClassification,
+    AutoModelForTokenClassification,
+    AutoTokenizer,
+    TFAutoModelForSequenceClassification,
+    TFAutoModelForTokenClassification,
+)
+
+from kiliautoml.utils.constants import ModelFramework, ModelRepository
+from kiliautoml.utils.helpers import get_last_trained_model_path, kili_print
+
+
+class HuggingFaceMixin(metaclass=ABCMeta):
+    """
+    Methods common to HuggingFace jobs
+    """
+
+    model_repository = ModelRepository.HuggingFace
+
+    @staticmethod
+    def _get_tokenizer_and_model(model_framework, model_path, model_type):
```

Comment:
Missing type hints. I believe typing is very important, especially when refactoring